audio_service/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Migrate to flutter_lints.
 * Queue messages from platform if init() called late.
 * Fix deep linking on Android (@vishna/@ryanheise).
+* Allow configuration to be updated after init (@ddfreiling).
```

Comment:
0.18.2 has already been released, so you will need to add a new version section for 0.18.3.